src/test/java/org/ays/runners/RunnerForRegressionTest.java

```diff
@@ -2,6 +2,9 @@
 
 import io.cucumber.testng.CucumberOptions;
 
-@CucumberOptions(tags = "@Regression")
+@CucumberOptions(
+        plugin = {"pretty"},
+        tags = "@Regression", publish = true
+)
```

Comment:
@cagla88 eğer burada ortak bir şey varsa bütün runner'larda çalışması için Abstract Class'a eklememiz daha iyi olabilir